eng/pipelines/templates/steps/nuget-cache.yml

```diff
@@ -0,0 +1,8 @@
+steps:
```

Comment:
Just to connect dots it looks like you copied this pretty closely from https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#netnuget which looks good to me.